db/schema.rb

```diff
@@ -55,6 +55,33 @@
   add_index "identities", ["session_uuid"], name: "index_identities_on_session_uuid", unique: true, using: :btree
   add_index "identities", ["user_id"], name: "index_identities_on_user_id", using: :btree
 
+  create_table "piis", force: :cascade do |t|
```

Comment:
http://peknet.com/blog/projects/two-hard-things

Since `verified` is a state of the data, I'd like to keep it out of the table name. `proofing_records` could work, though it's a little cumbersome. How about `facts` ?